File: src/model/card/Card.py

class Card:
	def __init__(self,name,effect):
		self.name = name
		self.effect = effect

	def xmlOutput(self):
		outStr = ""
		outStr +="<name>"
		outStr += self.name
		outStr +="</name>"
		return outStr

	def play(self,args):
		self.effect(args)

File: src/go/postprompt/card.go
package postprompt

type card struct {
	name string
	effect string
}

func NewCard(id int) (*card, error) {
	c := new(card)
	cardInfo, err := GetCardInfo(id)
	if err != nil { return nil, err }
	c.name = cardInfo[0]
	c.effect = cardInfo[1]
	return c, nil
}
